minikube/1.9/auditing/restart.sh

```diff
@@ -2,14 +2,12 @@
 
 set -xeou pipefail
 
-mkdir -p ~/.minikube/files
-curl -fsSL https://raw.githubusercontent.com/tamalsaha/tasty-kube/master/minikube/1.9/auditing/kube-apiserver.yaml > ~/.minikube/files/kube-apiserver.yaml
-curl -fsSL https://raw.githubusercontent.com/tamalsaha/tasty-kube/master/minikube/1.9/auditing/audit-policy.yaml > ~/.minikube/files/audit-policy.yaml
+mkdir -p ~/.minikube/files/audit
+curl -fsSL https://raw.githubusercontent.com/tamalsaha/tasty-kube/master/minikube/1.9/auditing/audit-policy.yaml > ~/.minikube/files/audit/audit-policy.yaml
```

Comment:
Why are you removing kube-apiserver.yaml ?

Reply:
We do not need this. We are not replacing original one with this one. Because that didn't work. Instead, we are editing with `vi` to add additional information.